package.json

```diff
@@ -71,6 +70,10 @@
     "user-home": "^2.0.0"
   },
   "devDependencies": {
+    "babel-polyfill": "^6.9.1",
+    "babel-preset-es2015": "^6.9.0",
+    "babel-runtime": "^6.9.2",
```

Comment:
Ah, I misunderstood and thought he was saying only the plugin wasn't necessary. Thanks for clarifying.